addons/packages/vsphere-cpi/metadata.yaml

```diff
@@ -8,8 +8,9 @@ spec:
   shortDescription: "vSphere Cloud Provider Interface"
   providerName: VMware
   maintainers:
-    - name: Andrew Sy Kim
-    - name: Lucheng Bao
-    - name: Nicole Han
+    - name: Lubron Zhan
+    - name: John McBride
+    - name: Nicholas Seemiller
```

Comment:
I'm not sure @seemiller and I should be added as maintainers explicitly in this package.

We are only apart of the @vmware-tanzu/pkg-vsphere-csi-owners group for admin operations.

Reply:
IC. let me remove